onoff.js

```diff
@@ -217,10 +217,9 @@ class Gpio {
       fs.write(this._valueFd, writeBuffer, 0, writeBuffer.length, 0, callback);
     } else {
       return new Promise((resolve, reject) => {
-        const writeBuffer = value === HIGH ? HIGH_BUF : LOW_BUF;
-        fs.write(this._valueFd, writeBuffer, 0, writeBuffer.length, 0, (err) => {
-          if (err) {
-            reject(err);
+        this.write(value, (error) => {
```

Comment:
All the code in onoff.js uses the name `err` rather than `error` for `Error` objects and I'd like to keep this consistent. Please rename `error` to `err`.